spec/controllers/subscription_spec.rb

```diff
@@ -20,7 +20,10 @@
     let(:subscription) do
       Routemaster::Models::Subscription.new(
         subscriber: 'charlie'
-      )
+      ).tap do |sub|
```

Comment:
This isn't ActiveRecord - `Subscription#initialize` doesn't support any other argument than `subscriber:`.
We _could_ refactor but this is a feature we're backporting to an old version of the bus (to facilitate upgrades), so I'm trying to keep changes minimal!